config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   resources :compliments, only: [:show]
   get 'complimentme' => 'compliments#index'
+  get 'family' => 'high_voltage/pages#show', id: 'family'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.